main.go

```diff
@@ -181,10 +181,17 @@ func main() {
 		logger.Fatal("new-route-services-server", zap.Error(err))
 	}
 
+	var metricsRegistry *mr.Registry
+	if c.Prometheus.Port != 0 {
+		metricsRegistry = mr.NewRegistry(log.Default(),
+			mr.WithTLSServer(int(c.Prometheus.Port), c.Prometheus.CertPath, c.Prometheus.KeyPath, c.Prometheus.CAPath))
```

Comment:
We should probably have some validation that all four items are set, since they're required in conjunction with each other. Does it make more sense to have the if statement here depend on a feature-enabling flag? in other words, if we don't have `PerAppPrometheusHttpMetricsReporting` set, what is the point of setting up the registry or checking config values?

Reply:
If PerAppPrometheusHttpMetricsReporting is set and prometheus isn't, then it fails during templating in the release. if you havn't set the certs, it'll fail when setting up the server. The cert path's themselves should be hardcoded if you have the port set in the release.

Reply:
The point of setting up a registry without per_app_http prom metrics is that you still get the default metrics. Whether that be more metrics which we incorperate into the new method of emitting metrics, or the default metrics(questionable) that are emitted by go-metrics-registry.